frameworks/kafka/docs/configure.md

```diff
@@ -194,6 +195,28 @@ The name of this Kafka instance in DC/OS. This is an option that cannot be chang
 *   **In DC/OS CLI options.json**: `name`: string (default: `kafka`)
 *   **DC/OS web interface**: The service name cannot be changed after the cluster has started.
 
+## Kill Grace Period
+
+The number of seconds to grant each broker to cleanly shutdown in response to SIGTERM.
+The option is termed `brokers.kill_grace_period`.
```

Comment:
Use the `brokers.kill_grace_period` configuration option to set a kill grace period.